packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -96,10 +96,12 @@ You would then get back an ID token as well as your access and refresh tokens. S
 }
 ```
 
-### Get a new access token/ID token silently for your SPA
+## Renew access and ID tokens with SPAs
 
-With a normal Single-Page Application (SPA), it is usually undesirable to redirect the user to a sign-in page during normal navigation. For example, a user could request access to a resource, prompting your SPA to send a request to the Okta `/authorize` endpoint. Normally, if a user doesn't have a valid session, this request results in a redirect to a sign-in page. To avoid this disruptive redirection, the endpoint allows for a request parameter called `prompt`. If the value of the `prompt` parameter is `none`, this guarantees that the user won't be prompted to sign in, regardless of whether they have an active session. Instead, your application either silently obtains the requested tokens or an OAuth error response occurs.
+With a SPA, it is usually undesirable to redirect the user to a sign-in page during normal navigation. To avoid this disruptive redirection, the `/authorize` endpoint allows the use of a request parameter called `prompt`. If the value of the `prompt` parameter is `none`, this guarantees that the user won't be prompted to sign in, regardless of whether they have an active session. Instead, your application either silently obtains the requested tokens or an OAuth error response occurs. Until now, the `prompt` parameter was the only way for a SPA to maintain user sessions without prompting the user to sign in multiple times.
```

Comment:
`Until now` - that's a bit vague, if this stays on our doc pages for years it'll be confusing and outdated info. Perhaps can you specify a bit more details on when "now" is?